modular_skyrat/modules/mob_container/mob_container.dm

```diff
@@ -0,0 +1,94 @@
+#define COMSIG_MOB_CONTAINER_ENTERED "mob_container_entered"
```

Comment:
don't define signals here

Reply:
These will be moved. Everything is being kept in one place for now since it's a draft.